source/AI.cpp

```diff
@@ -2432,20 +2435,21 @@ void AI::Attack(Ship &ship, Command &command, const Ship &target)
 	// Average distance from this ship's weapons to the enemy ship.
 	const double weaponDistanceFromTarget = direction.Length() - totalRadius / 3.;
 
-	// If this ship has mostly long-range weapons, or some weapons have a
+	// If this ship has an artillery AI, or some weapons have a
 	// blast radius, it should keep some distance instead of closing in.
 	// If a weapon has blast radius, some leeway helps avoid getting hurt.
-	if(minSafeDistance || (useArtilleryAI && shortestRange < weaponDistanceFromTarget))
+	if(minSafeDistance || (artilleryOverride && weaponDistanceFromTarget < shortestArtillery)
+		|| (useArtilleryAI && shortestRange < weaponDistanceFromTarget))
```

Comment:
why exactly is the check for useArtilleryAI different than artillery override?

Reply:
bc artillery override (this) forces using only the longest range weapons, whereas the battle AI does series of attacks

Reply:
ok sounds good